dev/tools/update_icons.dart

```diff
@@ -36,10 +36,17 @@ const Map<String, List<String>> _platformAdaptiveIdentifiers = <String, List<Str
 };
 
 const Map<String, String> _identifierRewrites = <String, String>{
+  '1x': 'one_x',
   '360': 'threesixty',
-  '3d_rotation': 'threed_rotation',
+  '2d': 'two_d',
+  '3d': 'threed',
```

Comment:
this inconsistency seems unfortunate (2d vs 3d)

Reply:
it does, though it can likely be subject to a future automated fix

Reply:
Can we avoid it by just calling the '2d' one "twod"? It's new here, right?

Reply:
You're right, there's no usage in g3 so it's safe to do. Fixed